cpp/source/deal.II/electrochemical_physics.h

```diff
@@ -49,7 +49,8 @@ class ElectrochemicalPhysics : public Physics<dim>
 {
 public:
   ElectrochemicalPhysics(
-      std::shared_ptr<PhysicsParameters<dim> const> parameters);
+      std::shared_ptr<PhysicsParameters<dim> const> parameters,
+      boost::mpi::communicator mpi_communicator);
```

Comment:
No. I think I did it just because at the beginning we pass the database and the communicator separately and I just kept the communicator alone everywhere.